.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,159 @@
+name: Bug Report
+description: Broken or unintended behavior with one of our libraries.
+labels: [bug-unconfirmed, question]
+body:
+- type: markdown
+  attributes:
+    value: |
+      ## Before Submitting your Issue
+      Please check to see if your question is answered in the FAQ for the library you are using:
+      - [@azure/msal-browser](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-browser/FAQ.md)
+      - [@azure/msal-node](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-node/FAQ.md)
+      - [@azure/msal-angular](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-angular/FAQ.md)
+      - [@azure/msal-react](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-react/FAQ.md)
+      - [@azure/msal or msal](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-core/docs/FAQ.md)
+      
+      If you have questions about timelines, please see our [roadmap](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/roadmap.md). Issues pertaining to timelines may be closed without comment.
+
+- type: markdown
+  attributes:
+    value: |
+      ## Issue Details
+    
+- type: dropdown
+  attributes:
+    label: Core Library
+    options: 
+      - "@azure/msal or msal"
+      - "@azure/msal-browser"
+      - "@azure/msal-node"
```

Comment:
```suggestion
      - "@azure/msal-node"
      - "passport-azure-ad"
      - "adal.js"
```

Reply:
I guess these can be added later once their PRs have been merged

Reply:
No harm in getting ahead of it. Done.